buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/RestTestsFromSnippetsTask.groovy

```diff
@@ -161,10 +161,25 @@ public class RestTestsFromSnippetsTask extends SnippetsTask {
             if (false == test.continued) {
                 current.println('---')
                 current.println("\"line_$test.start\":")
+                /* Since all these features *could* be required by any doc test
+                 * and we can't tell here if they *are* required by this doc
+                 * test we emit a skip all of them in case this is picked up by
+                 * a test runner that doesn't support them. */
```

Comment:
Right. The java runner is the only one running these. I had to edit this to add the skip for warnings or it wouldn't start after the changes I made. I figure from there I should add the others "just in case" another runner gets its hands on these. I can remove them if you think it is silly.